pom.xml

```diff
@@ -46,10 +46,6 @@
 				<repository>
 						<id>spigot-repo</id>
 						<url>https://hub.spigotmc.org/nexus/content/groups/public/</url>
-				</repository>
-								<repository>
-						<id>devoted-repo</id>
-						<url>http://dydoisbutts.info:8080/plugin/repository/everything/</url>
```

Comment:
I didnt change the name, but this is actually teals build server (as you can see in the domain) and it needs to stay.